README.md

```diff
@@ -1,8 +1,8 @@
-# Transbank Opencart 3.x Onepay Plugin
+# Transbank Opencart 3.x Webpay Plugin
 
 ## Descripción
 
-Este plugin de Opencart 3.x implementa el [SDK PHP de Onepay](https://github.com/TransbankDevelopers/transbank-sdk-php) en modalidad checkout. 
+Este plugin de Opencart 3.x implementa el [SDK PHP de Transbank](https://github.com/TransbankDevelopers/transbank-sdk-php) 
```

Comment:
Webpay?

Reply:
@leosoto If you refer to `SDK PHP de Transbank` I used Transbank instead of Webpay because we have an unique SDK for Transbank, but if you prefer I could change it to Webpay

Reply:
I just feel it's weird that the description of the webpay plugin doesn't say webpay anywhere. Maybe the whole description needs to be reworded.